Cargo.toml

```
@@ -25,4 +25,4 @@ repository = "carllerche/string"
 branch = "master"
 
 [dependencies]
-bytes = { version = "0.4", optional = true }
+bytes = { version = "0.5", optional = true }
```

```suggestion
bytes = { version = "0.4", optional = true }
bytes05 = { package = "bytes", version = "0.5", optional = true }
```

How about adding a new Cargo feature instead of replacing the existing one, so that this won't introduce a breaking change?